lib/unlocked/EditTile.dart

```diff
@@ -0,0 +1,485 @@
+                                InkResponse(
+                                  child: new Container(
+                                    width: 20,
+                                    height: 20,
+                                    decoration: new BoxDecoration(
+                                      color: Color(0xffEACA1E),
+                                      shape: BoxShape.circle,
+                                    ),
+                                  ),
+                                ),
+                                SizedBox(width: 30),
+                                InkResponse(
+                                  child: new Container(
+                                    width: 20,
+                                    height: 20,
+                                    decoration: new BoxDecoration(
+                                      color: Color(0xffF4828E),
+                                      shape: BoxShape.circle,
+                                    ),
+                                  ),
+                                ),
+                                SizedBox(width: 30),
+                                InkResponse(
+                                  child: new Container(
+                                    width: 20,
+                                    height: 20,
+                                    decoration: new BoxDecoration(
+                                      color: Color(0xffBA82DF),
+                                      shape: BoxShape.circle,
+                                    ),
+                                  ),
+                                ),
+                                SizedBox(width: 30),
+                                InkResponse(
+                                  child: new Container(
+                                    width: 20,
+                                    height: 20,
+                                    decoration: new BoxDecoration(
+                                      color: Color(0xffF99E5C),
+                                      shape: BoxShape.circle,
+                                    ),
+                                  ),
+                                ),
+                                SizedBox(width: 30),
+                                InkResponse(
+                                  child: new Container(
+                                    width: 20,
+                                    height: 20,
+                                    decoration: new BoxDecoration(
+                                      color: Color(0xffE53935),
+                                      shape: BoxShape.circle,
+                                    ),
+                                  ),
+                                ),
+                              ],
+                            ),
+                          ),
+                          onTap: () =>
+                              print('Tile Name'), // Debug: change to edit
```

Comment:
Instead of printing `Tile Name` shouldn't we be changing the color of the tile?

Reply:
I have not implemented this yet, maybe in the next PR